src/server/storage.ts

```diff
@@ -70,24 +53,13 @@ export async function fetchCodeManifest(info: MinimalJobInfo) {
 }
```

Comment:
I think this can be just `urlForFile` or something like that, we don't need the `content` part any longer, that was just for when files were stored in `tmp` in local dev